test/test_common.cpp

```diff
@@ -1,7 +1,7 @@
 #include "../cheri_compressed_cap.h"
-#include <unistd.h>
```

Comment:
?

Reply:
No longer needed since the catch include provides all the necessary definitions.

Reply:
I guess it was a workaround for it not being self-contained before then?.. Doesn't belong in a reformat commit though

Reply:
Yeah I've moved it into the correct commit now.